### PR TITLE
Fix anonymous user mixin error that occurs if user isn't logged in

### PR DESCRIPTION
**Pull request by shivam-kapila**
Fixes https://sentry.metabrainz.org/metabrainz/listenbrainz-beta/issues/99811/?environment=production&referrer=alert_email

### CODE REVIEW

File: listenbrainz/webserver/views/recommendations_cf_recording.py [hunk 1]
```diff
@@ -92,13 +92,17 @@ def _get_template(active_section, user):
...
-    spotify_data = spotify.get_user_dict(current_user.id)
+    spotify_data = {}
+    current_user_data = {}
 
-    current_user_data = {
-            "id": current_user.id,
-            "name": current_user.musicbrainz_id,
-            "auth_token": current_user.auth_token,
-    }
+    if current_user:
```
Comment (vansika) [inline listenbrainz/webserver/views/recommendations_cf_recording.py:98]: I think this will work `current_user.is_authenticated`

**Review by vansika — CHANGES_REQUESTED** [review]
Rn, the page still error out if the user is not logged in , the suggested change should work I think.

File: listenbrainz/webserver/views/recommendations_cf_recording.py (hunk 1 above)
Comment (vansika) [inline listenbrainz/webserver/views/recommendations_cf_recording.py:98]:
```suggestion
    if current_user.is_authenticated:
```

**Review by vansika — APPROVED** [review]

File: listenbrainz/webserver/views/recommendations_cf_recording.py (hunk 1 above)
Comment (shivam-kapila) [reply listenbrainz/webserver/views/recommendations_cf_recording.py]: yep

**Pull request merged**